client/checkout/platform-checkout/express-button/woopay-express-checkout-button.js

```diff
@@ -23,6 +43,13 @@ export const WoopayExpressCheckoutButton = ( {
 			return; // eslint-disable-line no-useless-return
 		}
 
+		wcpayTracks.recordUserEvent(
+			'platform_checkout_express_button_clicked',
```

Comment:
Should we use the event constants here instead?

Reply:
Absolutely. Good catch! Resolved in 1279fc30